### DISCUSSION TITLE
Duplicated primefaces version in href link

### DISCUSSION THREAD

**Discussion by MarqGrivkow** [primefaces/community#51] [PrimeFaces] [answered]
Hi,

I noticed that in page source I see duplicated parameteres v=13&v=13 ie:
... link type='text/css' rel='stylesheet' href='[/rms-gui/jakarta.faces.resource/theme.css.xhtml?ln=primefaces-nova-light&amp;v=13.0.0&amp;v=13.0.0](https://vortex-rms-gui-stg.sig-cloud.corpnet.pl/rms-gui/jakarta.faces.resource/theme.css.xhtml?ln=primefaces-nova-light&v=13.0.0&v=13.0.0)'...

It's attached to all the resources automatically generated on server side. ( jquery, components etc ) 

Is there a way to  deduplicate this? 

Marek

**Comment by melloware** [comment, marked as answer]
@MarqGrivkow you must have custom resource handler or somehow two PF jars?  if you look at the PF Showcase I only see this:

```xml
<link type="text/css" rel="stylesheet" href="/showcase/javax.faces.resource/theme.css.xhtml?ln=primefaces-saga&amp;v=13.0.0" />
```
And I am looking at all my PF 13 apps and I only see 1 `v=13.0.0` in all URLs

**Reply by jepsar** [reply]
Same here, but an additional one for PrimeFaces Extensions it seems:

http://localhost:8080/context/javax.faces.resource/primeicons/primeicons.css.xhtml?ln=primefaces&v=12.0.0&e=12.0.7

**Reply by melloware** [reply]
Correct extensions uses `e=` so I could tell the two versions apart it used to be `v=`.

**Reply by MarqGrivkow** [reply]
Hi,

Thanks for a hint with two jars and resource handler. I'm investigating it ( no succes as for now )  and will return soon.

**Comment by MarqGrivkow** [comment]
Hi,

Thank you again! You were right. The problem was in dependencies. I've been exporting additional jar in maven plugin.  

It solves my problem.

**Comment by MarqGrivkow** [comment]
As @melloware wrote - I checked dependencies and removed part responsible for exporting additional jar to lib.
